# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/crore/makefile-output.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Hare build seems to output the binary to crore-v0.2.2 so we overwrite to
simply -o crore to allow the rest of the makefile to work properly.

  - NoTag <notag@t2sde.org>

--- crore-v0.2.2/Makefile.vanilla	2024-01-29 11:57:57.544780665 +0100
+++ crore-v0.2.2/Makefile	2024-01-29 11:58:19.592049297 +0100
@@ -1,4 +1,4 @@
-VERSION=0.1.3
+VERSION=0.2.2
 PREFIX?=/usr
 BINDIR?=$(PREFIX)/bin
 MANDIR?=$(PREFIX)/share/man
@@ -7,7 +7,7 @@
 .DEFAULT_GOAL=all
 
 crore:
-	hare build
+	hare build -o crore
 
 crore.1: crore.1.scd
 	scdoc < $< > $@
